app/src/main/java/org/piramalswasthya/sakhi/repositories/AbhaIdRepo.kt

```
@@ -827,4 +832,96 @@ class AbhaIdRepo @Inject constructor(
+                    request = FaceAuthInitRequest(),
+                    requestId = generateUUID(),
+                    timestamp = getCurrentTimestamp()
+                )
+                if (response.isSuccessful) {
+                    val responseBody = response.body()?.string()
+                    val result = Gson().fromJson(responseBody, FaceAuthInitResponse::class.java)
+                    NetworkResult.Success(result)
+                } else {
+                    sendErrorResponse(response)
+                }
+            } catch (e: IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)